src/retry/client.rs

```diff
@@ -66,7 +67,7 @@ impl Deserializer for Json {
             .chain_err(|| "failed json deserialization")
     }
     fn content_type(&self) -> header::HeaderValue {
-        header::HeaderValue::from_static("text/json; charset=utf-8")
+        header::HeaderValue::from_static("application/json")
```

Comment:
This is clearly the correct form, but I can just hope that the previous form was not there to workaround some buggy cloud provider.

Reply:
Yeah, I had done some git history sleuthing around this. The original commit introducing this is 4b217bd71bc01a79dd01e1d89f4c998cfa5ed4a3, which... looks like this string might've just been adjusted from the `Raw` version of this block?